### PR TITLE
Dse documentation

### PR DESCRIPTION
**Pull request by ShvetankPrakash**
Added README documentation to explain how to use dse proj template.

### CODE REVIEW

**Comment by ShvetankPrakash** [comment]
FYI @tcal-x looking [here](https://github.com/google/CFU-Playground/tree/dse_documentation/proj/dse_template) at the README visually may be easier to check than the raw markdown.

**Comment by ShvetankPrakash** [comment]
Rebased @tcal-x FYI

File: proj/dse_template/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+2. Install the VexRiscv Build Tools
+- Run `$CFU_ROOT/scripts/setup_vexriscv_build.sh`
+
+3. Install Vizier 
+- Run `$CFU_ROOT/scripts/setup_vizier.sh`
+
+4. Test DSE 
+- In `proj/dse_template` run `python vizier_dse.py`
+- Sample output should look like this (with different raw values):
```
Comment (tcal-x) [inline proj/dse_template/README.md:18]: Can you add a note about how long it will take to get to this point?

Comment (ShvetankPrakash) [reply proj/dse_template/README.md:19]: @tcal-x added :)

**Comment by tcal-x** [comment]
@ShvetankPrakash  I see this instead of what's in the README:

```
INFO:SoC:--------------------------------------------------------------------------------
INFO:SoC:Initializing ROM rom with contents (Size: 0x5f70).
INFO:SoC:Auto-Resizing ROM rom from 0x20000 to 0x5f70.
<long pause>
NUMBER OF CYCLES: 205915399.0
NUMBER OF CELLS:  22071

EXITING DSE...

```

**Comment by ShvetankPrakash** [comment]
> @ShvetankPrakash I see this instead of what's in the README:
> 
> ```
> INFO:SoC:--------------------------------------------------------------------------------
> INFO:SoC:Initializing ROM rom with contents (Size: 0x5f70).
> INFO:SoC:Auto-Resizing ROM rom from 0x20000 to 0x5f70.
> <long pause>
> NUMBER OF CYCLES: 205915399.0
> NUMBER OF CELLS:  22071
> 
> EXITING DSE...
> ```

After this "EXITING DSE" there is no output returned from Vizier? Does the program terminate? @tcal-x

**Review by tcal-x — APPROVED** [review]
LGTM

**Pull request merged**